index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./css/styles.css">
+    <link rel="stylesheet" type="text/css" href="./css/projects.css">
+    <link rel="stylesheet" type="text/css" href="./css/about.css">
+    <title>Profolio</title>
+</head>
+
+<body>
+
+    <section class="container">
+        <header>
+            <nav class="Header">
+
+                <ul class="header-logo">
+                    <li class="logo"> My Logo </li>
+                    <li class="hamburg-logo"><img src="Img/Hlogo.png" alt="hamburger icon"> </li>
+                </ul>
+            </nav>
+        </header>
+        <div class="section-header">
```

Comment:
As highlighted by the previous reviewer, kindly change this tag fro. the general `div` to a `section` tag. It's important to use the correct semantics tags so as not to confuse screen readers.

Reply:
This is the structure the previous reviewer asked me to follow. You are saying I should remove the div. After another reviewer will come and tell me to change yours again
<section>
  <header>
    <nav>
      <a> your logo </a>
      <ul>
        <li> hamburger icon </li>
      </ul>
    </nav>
  </header>
  <div>
  <h1> here are the tag elements for your name </h1>
  <p> next a `p` for your presentation </p>
  </div>
  <div>
    <a> for your let's connect </a>
    <ul> for your list of social icons
      <li> your list of social icons </li>
    </ul>
  </div>
</section>

Reply:
<section>
  <header>
    <nav>
      <a> your logo </a>
      <ul>
        <li> hamburger icon </li>
      </ul>
    </nav>
  </header>
  <div>
  <h1> here are the tag elements for your name </h1>
  <p> next a `p` for your presentation </p>
  </div>
  <div>
    <a> for your let's connect </a>
    <ul> for your list of social icons
      <li> your list of social icons </li>
    </ul>
  </div>
</section>